app/admin/requests/Staff.php

```diff
@@ -25,7 +25,7 @@ public function rules()
                 ['language_id', 'admin::lang.staff.', 'integer'],
                 ['staff_role_id', 'admin::lang.staff.label_role', 'required|integer'],
                 ['groups', 'admin::lang.staff.label_group', 'required|array'],
-                ['locations', 'admin::lang.staff.label_location', 'nullable|array'],
+                ['locations', 'admin::lang.staff.label_location', 'required|nullable|array'],
```

Comment:
we don't need the `required` rule here.. 

Add `nullable` rule to `langugage_id` field and fix the lang string, it should be `admin::lang.staff.label_language_id`